uwblueprint/jump-math
# Sort Admin Users Table

## Notion ticket link

[Sort Admin Users](https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=5b4728f1da0c4df187f8e1c14785265b&pm=s)



## Implementation description
* Adding a modal to sort the admin user table
Figma Link: https://www.figma.com/file/cRjJxm8087kkymfH4ieKau/Platform-Working-File?node-id=486%3A18351&t=HoaK1YOnhwFX5UPR-0


## Steps to test
1. Ensure that Ascending and Descending option works
2. Ensure that sort is by property selected
3. Ensure that component looks like design



## What should reviewers focus on?
* 
<img width="9" alt="Screen Shot 2022-12-04 at 9 56 45 PM" src="https://user-images.githubusercontent.com/87676999/205540309-c0c2aa80-3bff-473c-887f-e4c61037c017.png">
<img width="1725" alt="Screen Shot 2022-12-04 at 9 56 54 PM" src="https://user-images.githubusercontent.com/87676999/205540316-057eae85-75b6-44dc-931b-6f1f29f430eb.png">


## Checklist
